taxonomies/ingredients.txt

```diff
@@ -47752,15 +47803,23 @@ fr:céleri rave râpé
 <en:celeriac
 fr:jus de céleri-rave concentré, jus de céleri rave concentré, jus concentré de céleri rave
 
+<en:root vegetable
 en:taro tuber
 fr:Tubercule de taro
+ja:タロイモ
+wikidata:en:Q1771944
 ciqual_food_code:en:53200
 ciqual_food_name:en:Taro, tuber, raw
 ciqual_food_name:fr:Taro, tubercule, cru
 
+<en:taro tuber
```

Comment:
```suggestion
<en:taro tuber
en:taro
```

Reply:
Is that right? I feel like "taro tuber" would be a part of "taro." I also don't think adding (or removing) "tuber" should change the translation like that. Not sure if there's a distinguishable English translation for サトイモ.

Reply:
From my understanding (wikipedia page I can understand is the french one (no wikipedia page in English) and is rather incomplete):
taro tuber (タロイモ) -> https://www.wikidata.org/wiki/Q1771944 -> root of the plant
taro (サトイモ) -> https://www.wikidata.org/wiki/Q227997 -> the plant

Hence, I would do something like this (it seems to be coherant with your comment):
```

en:Taro
af:Taro
am:ጎደሬ
ar:قلقاس مأكول
be:Colocasia esculenta
bh:अरुई
bi:Taro
bn:কচু
ca:Taro
cs:Kolokázie jedlá
de:Taro
dv:އޮޅު އަލަ
el:Κολοκασία η εδώδιμος
eo:Taro
es:Colocasia esculenta
eu:Taro
fa:تارو
fi:Taaro
fj:Dalo
fr:Colocasia esculenta
gl:Taro
gu:અળવી
ha:Gwaza
he:טארו
hi:अरबी सब्जी
hr:Taro
ht:Chou Karayib
hu:Taró
id:Talas
ig:Ede
it:Colocasia esculenta
ja:サトイモ, 里芋, さといも
ka:ტარო 
kn:ಕೆಸು
ko:토란
la:Colocasia esculenta
lt:Valgomoji kolokazija
lv:Kolokāzija
mg:Saonjo
ml:താള്
mr:अळू
ms:Keladi Cina
ne:कर्कलो
nl:Taro
nn:Taro
no:Taro
pa:ਅਰਵੀ
pl:Kolokazja jadalna
pt:Taro
rn:Amateke
ru:Таро
sg:Langä
sl:Taro
sm:Talo
sr:Таро
su:Kimpul
sv:Taro
sw:Magimbi
ta:சேம்பு
th:เผือก
tl:Gabi
to:Talo
tr:Gölevez
ty:Taro
uk:Таро
ur:تارو
vi:Colocasia esculenta
za:Biek
zh:芋
zu:Amadumbe
wikidata:en:Q227997


<en:root vegetable
<en:Taro
en:taro tuber
ar:قلقاس
de:taro
es:taro
fr:Taro, Tubercule de taro
it:taro
ja:タロイモ
ru:Таро
vi:Khoai môn
wikidata:en:Q1771944
ciqual_food_code:en:53200
ciqual_food_name:en:Taro, tuber, raw
ciqual_food_name:fr:Taro, tubercule, cru
```

Reply:
I meant to ask something like like "wouldn't 'taro tuber' mean the tuber of taro, so it would be a subcategory?"

My understanding on is that タロイモ and taro may refer to a family of closely related vegetables, taro could also refer to a specific plant, and サトイモ is a vegetable belonging to the broader version of taro but not the same as the specific version of taro. I don't know much about taro, so I'm not certain that this is accurate.

Reply:
Unfortunately, I do not know much as well. 

From https://ja.wikipedia.org/wiki/%E3%82%BF%E3%83%AD%E3%82%A4%E3%83%A2:
> サトイモ、エビイモ、エグイモ、カラノイモ、タケノコイモ、タイモもタロイモの一種であり、サトイモはタロイモのうち最も北方で栽培されている品種群である。 

Hence, サトイモ (Q227997) seems to be part of タロイモ (Q1771944). The opposite of what I wrote in last message. 
Additionally, also wrong in my last message, both of them are roots, right?


To make category and sub-category, we use the "<" sign. 

Parent should be (child of en:root vegetable):
```
<en:root vegetable
en:taro tuber
etc.
```

Child (sub-category of en:taro tuber) should be (it start by "<en:taro tuber" to indicate that en:Taro is a child of en:taro tuber):
```
<en:taro tuber
en:Taro
etc.
```

Reply:
I agree with サトイモ being a child of タロイモ.

Wouldn't en:taro tuber be a child of en:Taro? I feel like a part of a plant should be a child of the whole plant. The English Wikipedia page says people eat the tuber and leaves of taro.

Reply:
Yes, it make sense.

Reply:
So in conclusion, we should have the following (plus other languages), am I correct?
```
<en:plant
en:Taro
ja:タロイモ

<root vegetable
<en:Taro
en:taro tuber

<en:Taro
ja:サトイモ
```

Reply:
yes! correct

Reply:
[:)](https://github.com/Naruyoko/openfoodfacts-server/commit/a8ad6a6f446d708ca22ffe07ff1e2b1e6328c83f)